pootle/templates/browser/index.html

```diff
@@ -149,7 +149,11 @@ <h3 class="top">{% trans "Updates" %}</h3>
     refreshed automatically in <strong>30 seconds</strong>.
     {% endblocktrans %}
   {% elif user.is_superuser %}
-    {% trans "Stats are out of date. See documentation for dealing with stats issues." %}
+    {% blocktrans trimmed %}
+    Stats are out of date.
+    See <a href='http://docs.translatehouse.org/projects/pootle/en/{{ rtd_version }}/server/rq.html'>documentation</a>
```

Comment:
Instead of hardcoding doc URLs, you can retrieve them in Python via `get_docs_url('server/rq.html')` from `pootle.core.utils.docs`.